x-pack/plugins/fleet/server/services/epm/elasticsearch/template/install.ts

```diff
@@ -341,7 +341,7 @@ export async function ensureDefaultComponentTemplate(
     await putComponentTemplate(esClient, logger, {
```

Comment:
`putComponentTemplate` is not returning a promise, but an object containing `clusterPromise`